{{cookiecutter.project_slug}}/home/api/v1/permissions.py

```diff
@@ -0,0 +1,22 @@
+import os
+import hmac
+
+from rest_framework.permissions import BasePermission
+
+
+class CrowboticsExclusive(BasePermission):
+    def has_permission(self, request, view):
+        header_signature = request.META.get("HTTP_CB_SIGNATURE", None)
```

Comment:
Shouldn't this be a standard token header?

Reply:
Nope, cuz we only want us to access this resource from the CB dashboard. Any logged-in user with a bearer token could access it if we use a standard token.

Reply:
I get your point. It should be `HTTP_X_CB_SIGNATURE` then since it's standard to add X to the front of arbitrary headers.

Reply:
Done 
dbceefd